src/balena.sbvr

```diff
@@ -528,21 +529,6 @@ Fact type: device has ip address
 	Necessity: each device has at most one ip address
 Fact type: device has mac address
 	Necessity: each device has at most one mac address
-Fact type: device has memory usage
-	Necessity: each device has at most one memory usage
-Fact type: device has memory total
-	Necessity: each device has at most one memory total
-Fact type: device has storage block device
-	Necessity: each device has at most one storage block device
-Fact type: device has storage usage
-	Necessity: each device has at most one storage usage
-Fact type: device has storage total
-	Necessity: each device has at most one storage total
-Fact type: device has cpu usage
-	Necessity: each device has at most one cpu usage
-Fact type: device has cpu temp
-	Necessity: each device has at most one cpu temp
-Fact type: device is undervolted
```

Comment:
I don't think this can be safely removed until the async migration relying on it is finalized